### ISSUE TITLE
File upload subtask: Frontend can request and display files

### ISSUE THREAD

**Issue by cjhskippy** [CMSgov/cms-carts-seds#793]
Follows #792 . Provides the user with a way to see files that have been uploaded.

- [ ] User experience in client software includes AC from #408 
- [ ] Once upload is complete, user can see the file name.  Providing a link to remove the file is out of scope for this issue.
- [ ] User has an option to download the file, 
- [ ] Client provides an error message to the user if the upload fails (session expired / backend doesn't think you have permission / S3 upload failed)

- [ ] Frontend component knows how to request / display files

**Comment by TaraWilliamsPGH** [comment]
Covered by ticket 815.
